concordium-node/README.md

```diff
@@ -111,11 +111,25 @@ Environment variables only apply to the default build. This links with shared Ha
   [concordium-consensus](../concordium-consensus/) dependency is the same regardless of how the
   node itself is built, the `--release` only applies to the optimization of the rust node components.
   
-Documentation about what <CONCORDIUM_NODE_CONFIG_DIR> and <CONCORDIUM_NODE_DATA_DIR> is seen at https://github.com/Concordium/concordium-node/blob/main/VARIABLES.md#common
+Documentation about what `<CONCORDIUM_NODE_CONFIG_DIR>` and `<CONCORDIUM_NODE_DATA_DIR>` is seen at https://github.com/Concordium/concordium-node/blob/main/VARIABLES.md#common
 
 - The node built with Haskell library auto-discovery is not suitable for distribution to other
   machines. It is a dynamically linked binary with a large number of shared library dependencies.
 
+### Genesis Files
+Running the node on a local machine would requires the *genesis files* to be set up in the `<CONCORDIUM_NODE_DATA_DIR>`.
```

Comment:
More specifically, this is for running starting a local chain. To run the node against either mainnet or testnet you would need to get the relevant genesis.dat file